styles.css

```diff
@@ -4,17 +4,13 @@
     background-image: url('images/background.jpg');
 }
 
-h1, h2,h3,h4, h5, h6, p,form,div,table{
+h1, h2,h3,h4, h5, h6, p,form,div,table,title{
```

Comment:
You should put the `font-family` on `html` `body` (rather than listing every single type of element here)